Quick workaround to fix build with USE=sql for Qt6.6 due to [1]
while waiting for PyQt6-6.6.0. Not a proper fix but nothing in
tree needs this bit currently.

USE=bluetooth is still broken, but is not essential (no revdeps),
and can instead be masked for the time being.

[1] https://github.com/qt/qtbase/commit/d924253c91
--- a/sip/QtCore/QtCoremod.sip
+++ b/sip/QtCore/QtCoremod.sip
@@ -24,3 +24,3 @@
 
-%Timeline {Qt_6_0_0 Qt_6_1_0 Qt_6_2_0 Qt_6_3_0 Qt_6_4_0 Qt_6_5_0}
+%Timeline {Qt_6_0_0 Qt_6_1_0 Qt_6_2_0 Qt_6_3_0 Qt_6_4_0 Qt_6_5_0 Qt_6_6_0}
 
--- a/sip/QtSql/qsqlresult.sip
+++ b/sip/QtSql/qsqlresult.sip
@@ -65,3 +65,5 @@
     int boundValueCount() const;
+%If (- Qt_6_6_0)
     QList<QVariant> &boundValues() const;
+%End
     QString executedQuery() const;
